packages/components/forms/src/textarea/README.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: 'Textarea'
+type: 'component'
+status: 'stable'
+slug: /components/textarea/
+github: 'https://github.com/contentful/forma-36/tree/master/packages/components/forms/src/textarea'
+storybook: 'https://f36-storybook.contentful.com/?path=/story/form-elements-textarea--basic'
+typescript: './TextArea.tsx'
+---
+
+`Textarea` is a form component that allows the user to enter a sizeable amount of multi-line plain text.
+
+## Table of contents
+
+- [How to use Textarea](#how-to-use-textarea)
+- [Examples](#examples)
+  - [isRequired](#isrequired)
+  - [isDisabled](#isdisabled)
+  - [isInvalid](#isinvalid)
+  - [Using ref](#using-ref)
+- [Accessibility](#accessibility)
+
+## How to use Textarea
+
+Textarea can be imported from `@contentful/f36-components` or `@contentful/f36-forms`.
+This component is renders a form element, therefore it should be used within the context of a `<Form>`.
+
+## Examples
+
+```tsx
+<Textarea id="textarea" name="description" label="description" />
+```
+
+### isRequired
+
+```tsx
+<Textarea id="textarea" name="description" label="description" isRequired />
+```
+
+By using the `isRequired` prop, it’s possible to control the required state of the `textarea`.
+This prop will set `disabled` attribute of the HTML tag to `true`.
+
+### isDisabled
+
+```tsx
+<Textarea id="textarea" name="description" label="description" isDisabled />
+```
+
+By using the `isDisabled` prop, it’s possible to control the disabled state of the `textarea`.
+This prop will handle the disabled styles and set `disabled` attribute of the HTML tag to `true`.
+
+### isInvalid
+
+```tsx
+<Textarea id="textarea" name="description" label="description" isInvalid />
+```
+
+By using the `isInvalid` prop, it’s possible to control the error state of the `textarea`.
+This prop will handle the error styles.
+
+### Using ref
+
+```jsx static=true
+const textareaRef = React.createRef()
+
+<Textarea
+  id="textarea"
+  name="textarea"
+  label="My textarea"
+  placeholder="Placeholder"
+  ref={textareaRef}
+/>
+
+<Button onClick={() => textareaRef.current.focus()}>
+  Focus Textarea with ref
+</Button>
+```
+
+The `Textarea` is exposing the ref of the `textarea` tag as a prop,
+which means that you can pass a ref created with `React.createRef()` to it and it will work as expected.
+
+## Accessibility
+
+- To ensure accessibility, provide a `name` prop
+- In case the input is required, use `isRequired` prop
+- Provide validation to the input and use `isInvalid` to control the error state of `Textarea`. Also show meaninful error messages with the `ValidationMessage` component.
```

Comment:
Maybe the part of the `ValidationMessage` would be better off on the `FormControl` component??
As it's not strictly necessary to have a validation for using the textarea by itself, but it should be good to have if it's being used on a Form, but not sure how to phrase that. 🤔

Reply:
Yep! This is meant to be only recommendations for a11y forms
And I'm recommending them to use `ValidationMessage` because it is red and has the `!` icon
I mean it will look like the designs

But they should do that in FormControl of couse


Ah do you mean move this orientation to FormControl’s documentation? 
That's fine for me

Reply:
yeah, exactly having this part on the FormControl documentation, but we can have on both actually.

Reply:
I will leave it here then and will try to write something about this being relevant when the textarea is in a form then 👌